src/components/workspace/resize-panel-divider.scss

```diff
@@ -0,0 +1,84 @@
+@import "../vars.sass";
+
+.divider-container {
+  width: 4px;
+  height: calc(100vh - (#{$header-height} + #{$workspace-content-margin}));
```

Comment:
Is there a reason to use `vh` here rather than `100%`? Seems like that could result in the divider exceeding the height of its container.